NitroxLauncher/NitroxLauncher.csproj

```diff
@@ -28,6 +28,11 @@
       <ProjectReference Include="..\NitroxPatcher\NitroxPatcher.csproj" />
       <ProjectReference Include="..\NitroxServer-Subnautica\NitroxServer-Subnautica.csproj" />
     </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="Microsoft.VisualBasic" />
+      <Reference Include="PresentationFramework.Aero2" />
```

Comment:
Why do we need aero2 framework ?

Reply:
That was added accidentally while I was experimenting with stuff and I forgot to remove it.

Reply:
Well, VS yelled at me when I removed it. Apparently, the new Dropbox style I created needs it